src/components/multi-action-button/multi-action-button.style.ts

```diff
@@ -100,14 +100,6 @@ const StyledButtonChildrenContainer = styled.div<StyledButtonChildrenContainerPr
       text-align: ${align};
       z-index: ${theme.zIndex.overlay};
 
-      /* Styling for Safari. */
```

Comment:
**praise**: nice job finding out this can be replaced with a browser agnostic solution 👍🏼